projects/packages/jetpack-mu-wpcom/src/features/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -0,0 +1,634 @@
+					? '//widgets.wp.com/wpcom-block-editor/calypso.editor.css?minify=false'
+					: '//widgets.wp.com/wpcom-block-editor/calypso.editor.min.css',
+				array(),
+				$version
+			);
+		}
+	}
+
+	/**
+	 * Enqueues the WordPress.com block editor integration assets for both editor and front-end.
+	 */
+	public function enqueue_block_assets() {
+		// These styles are manually copied from //widgets.wp.com/wpcom-block-editor/default.view.css in order to
+		// improve the performance by avoiding an extra network request to download the CSS file on every page.
+		wp_add_inline_style( 'wp-block-library', '.has-text-align-justify{text-align:justify;}' );
+	}
+
+	/**
+	 * Determines if the current $post contains a justified paragraph block.
+	 *
+	 * @return boolean true if justified paragraph is found, false otherwise.
+	 */
+	public function has_justified_block() {
+		global $post;
+		if ( ! $post instanceof WP_Post ) {
+			return false;
+		}
+
+		if ( ! has_blocks( $post ) ) {
+			return false;
+		}
+
+		return str_contains( $post->post_content, '<!-- wp:paragraph {"align":"justify"' );
+	}
+
+	/**
+	 * Register the Tiny MCE plugins for the WordPress.com block editor integration.
+	 *
+	 * @param array $plugin_array An array of external Tiny MCE plugins.
+	 * @return array External TinyMCE plugins.
+	 */
+	public function add_tinymce_plugins( $plugin_array ) {
+		if ( $this->is_iframed_block_editor() ) {
+			$debug = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG;
+
+			$plugin_array['gutenberg-wpcom-iframe-media-modal'] = add_query_arg(
+				'v',
+				gmdate( 'YW' ),
+				$debug
+					? '//widgets.wp.com/wpcom-block-editor/calypso.tinymce.js?minify=false'
+					: '//widgets.wp.com/wpcom-block-editor/calypso.tinymce.min.js'
+			);
+		}
+
+		return $plugin_array;
+	}
+
+	/**
+	 * Ensures the authentication cookies are designated for cross-site access.
+	 */
+	private function enable_cross_site_auth_cookies() {
+		/**
+		 * Allow plugins to disable the cross-site auth cookies.
+		 *
+		 * @since 8.1.1
+		 *
+		 * @param false bool Whether auth cookies should be disabled for cross-site access. False by default.
+		 */
+		if ( apply_filters( 'jetpack_disable_cross_site_auth_cookies', false ) ) {
+			return;
+		}
+
+		add_action( 'set_auth_cookie', array( $this, 'set_samesite_auth_cookies' ), 10, 5 );
+		add_action( 'set_logged_in_cookie', array( $this, 'set_samesite_logged_in_cookies' ), 10, 4 );
+		add_filter( 'send_auth_cookies', array( $this, 'maybe_send_cookies' ), 9999 );
+	}
+
+	/**
+	 * Checks if we've stored any cookies to send and then sends them
+	 * if the send_auth_cookies value is true.
+	 *
+	 * @param bool $send_cookies The filtered value that determines whether to send auth cookies.
+	 */
+	public function maybe_send_cookies( $send_cookies ) {
+
+		if ( ! empty( $this->set_cookie_args ) && $send_cookies ) {
+			array_walk(
+				$this->set_cookie_args,
+				function ( $cookie ) {
+					call_user_func_array( 'jetpack_shim_setcookie', $cookie );
```

Comment:
This function lives [inside the jetpack plugin](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/functions.cookies.php#L21) so we will need to either copy it into the **mu-wpcom** package or make sure we use PHP version7.3+ in Simple and WoA and use `set_cookie`.

Reply:
`set_cookie` may be your best bet at this point, since Atomic runs at least 7.4 afaik, and simple now runs 8.1.